lib/getNFTInfo.ts

```diff
@@ -16,12 +25,15 @@ export interface GetNFTInfoResponse {
 export async function getNFTInfoFromTokenInfo(
   tokenId: ethers.BigNumber,
   tokenURI: string,
+  network: SupportedNetwork,
   forceImage: boolean = false,
 ): Promise<GetNFTInfoResponse | null> {
   const isDataUri = tokenURI.startsWith('data:');
   try {
     const tokenURIRes = await fetch(
-      isDataUri ? tokenURI : `/api/nftInfo/${encodeURIComponent(tokenURI)}`,
+      isDataUri
+        ? tokenURI
+        : `/api/network/${network}/nftInfo/${encodeURIComponent(tokenURI)}`,
```

Comment:
does the network matter for the nftInfo API? i know we should keep network to stay uniform, just a question out of curiousity

Reply:
At present it doesn't (and conceivably we may replace this API in the future either with Center or something else)